diode-server/proto/reconciler/v1/reconciler.proto

```diff
@@ -29,6 +30,7 @@ message RetrieveIngestionDataSourcesRequest {
     max_len: 255
   }];
   string sdk_version = 3 [(validate.rules).string = {pattern: "^(\\d)+\\.(\\d)+\\.(\\d)+$"}];
+  diode.v1.Entity entity = 4;
```

Comment:
should we remove it?

Reply:
yes! good spot, thanks

Reply:
actually there also shouldn't be diode-server/proto dir anymore